src/torchcodec/samplers/_implem.py

```diff
@@ -0,0 +1,62 @@
+from typing import List
+
+import torch
+
+from torchcodec.decoders import (  # TODO: move FrameBatch to torchcodec.FrameBatch?
```

Comment:
@ahmadsharif1 @scotts  Thoughts?

Reply:
I'll do it. We'll have to move the `FrameBatch` implementation within `torchcodec._something`, otherwise this is creating cyclic imports. I'll create a PR to tackle that immediately after this one is merge. I prefer not doing it here to avoid complicating the PR and the reviews.